helm/opa-mutator-app/rules/rules_awscontrolplane.rego

```diff
@@ -23,12 +23,13 @@ deny[msg] {
     msg = "Invalid choice of Master Node Instance Type"
 }
 
-
-# User has selected the same availability zone twice
+# User has selected the same availability zone twice. this should only be possible
+# in regions with limited AZs
 deny[msg] {
```

Comment:
So with this we only allow using the same AZ twice when there are not enough AZs available. We can also drop it completely and let people be able to choose whatever they like.